applications/leetcoin.md

```diff
@@ -107,81 +185,81 @@ Please also provide the GitHub accounts of all team members. If they contain no
 
 ## Development Status :open_book:
 
-We have created a landing page https://leetcoin.co and some initial backend skeleton structure https://github.com/greenoakxyz/leetcoin-app. There is no prototype yet and we are currently developing it.
+We have created a landing page https://leetcoin.co and some initial backend skeleton
+structure https://github.com/greenoakxyz/leetcoin-app. There is no prototype yet and we are currently developing it.
 
 ## Development Roadmap :nut_and_bolt:
 
 ### Overview
 
-- **Total Estimated Duration:** 20 Weeks
+- **Total Estimated Duration:** 37 Weeks
 - **Full-Time Equivalent (FTE):** 3 FTE
 - **Total Costs:** 29,999 DAI
 
 ### Milestone 1 — Implement Ink! Problem Builder, Solution and Compiler
 
-- **Estimated duration:** 8 weeks
+- **Estimated duration:** 21 weeks (Completion by March 15, 2023)
```

Comment:
Unfortunately, due to unforeseen circumstances, we have had to postpone the start of the project to January 2023. We understand that this delay may cause some inconvenience and we apologize for any issues this may cause. The revised completion dates are included in estimate durations